inc/sainexthopgroup.h

```diff
@@ -81,6 +115,37 @@ typedef enum _sai_next_hop_group_attr_t
     SAI_NEXT_HOP_GROUP_ATTR_TYPE,
 
     /**
+     * @brief Events that can initiate switchover from primary to backup next hop
+     *
+     * Can be a combination of the following:
+     * SAI_NEXT_HOP_SWITCHOVER_TRIGGER_NULL,
+     * SAI_NEXT_HOP_SWITCHOVER_TRIGGER_BFD,
+     * SAI_NEXT_HOP_SWITCHOVER_TRIGGER_PORT_DOWN
+     *
+     * @type sai_uint8_t
```

Comment:
Can you explain more? I not attended to that meeting so i don't have context here